noxfile.py

```diff
@@ -44,13 +44,28 @@
 reset_coverage = True
 
 
+def lock_deps(session: nox.Session):
+    args = ["pdm", "lock", "-G:all", "-dG:all"]
+    output = session.run_always(
+        *args,
+        "--check",
+        external=True,
+        success_codes=list(range(0, 128)),
+        silent=True,
+    )
+    if output:
```

Comment:
nox raises a `CommandFailed` exception if the return code is nonzero, would be better to go off of that instead of checking `if output:`, which seems unstable

Reply:
Oh dope

Reply:
As CommandFailed isn't documented, I think I'm just going to catch Exception instead.

Reply:
[`1cdbc44` (#1022)](https://github.com/DisnakeDev/disnake/pull/1022/commits/1cdbc4440b3e76cd343a3094bc5a63e3f8c386b7)